Fix build failure with tcl 8.6

With this patch, opencascade still builds with tcl 8.5.

Submitted upstream: http://www.opencascade.org/org/forum/thread_20125/

Index: opencascade/ros/src/Draw/Draw_Window.cxx
===================================================================
--- opencascade.orig/ros/src/Draw/Draw_Window.cxx
+++ opencascade/ros/src/Draw/Draw_Window.cxx
@@ -70,7 +70,7 @@
       errChannel = Tcl_GetStdChannel(TCL_STDERR);
       if (code != TCL_OK) {
         if (errChannel) {
-          Tcl_Write(errChannel, Interp->result, -1);
+          Tcl_Write(errChannel, Tcl_GetStringResult(Interp), -1);
           Tcl_Write(errChannel, "\n", 1);
         }
         Tcl_AddErrorInfo(Interp,
@@ -1044,7 +1044,7 @@
   mainWindow =
   Tk_MainWindow(interp) ;
   if (mainWindow == NULL) {
-    fprintf(stderr, "%s\n", interp->result);
+    fprintf(stderr, "%s\n", Tcl_GetStringResult(interp));
     exit(1);
   }
   Tk_Name(mainWindow) =
@@ -2064,7 +2064,7 @@
     OCC_CATCH_SIGNALS
     Standard_Integer res = Tk_Init(interp) ;
     if (res != TCL_OK)
-      cout << "tkLoop: error in Tk initialization. Tcl reported: " << interp->result << endl;
+      cout << "tkLoop: error in Tk initialization. Tcl reported: " << Tcl_GetStringResult(interp) << endl;
 
   } catch  (Standard_Failure) {
     cout <<"tkLoop: exception in TK_Init "<<endl;
@@ -2074,7 +2074,7 @@
   mainWindow =
     Tk_MainWindow(interp) ;
   if (mainWindow == NULL) {
-        fprintf(stderr, "%s\n", interp->result);
+        fprintf(stderr, "%s\n", Tcl_GetStringResult(interp));
         cout << "tkLoop: Tk_MainWindow() returned NULL. Exiting..." << endl;
         Tcl_Exit(0);
    }
